march_hardware/src/IMotionCube.cpp

```diff
@@ -97,10 +97,11 @@ void IMotionCube::writeInitialSettings(uint8 ecatCycleTime)
 
 void IMotionCube::actuateRad(float targetRad)
 {
-  if (std::abs(targetRad - this->getAngleRad()) > 0.2)
+  if (std::abs(targetRad - this->getAngleRad()) > 0.05)
```

Comment:
0.05 is probably okay, should be tested on the exoskeleton

Reply:
```suggestion
  if (std::abs(targetRad - this->getAngleRad()) > 0.075)
```